docs/tempo/website/getting-started/example-demo-app.md

```diff
@@ -21,9 +22,9 @@ The [docker-compose examples](https://github.com/grafana/tempo/tree/main/example
 
 Some of the examples include:
 
-- Trace discovery with Loki
-- Basic Grafana Agent/OpenTelemetry Setup
-- Various Backends (S3/GCS/Azure)
+* Trace discovery with Loki
+* Basic Grafana Agent/OpenTelemetry Setup
+* Various Backends (S3/GCS/Azure)
```

Comment:
I think this change goes against the recently published guidelines in the writer's toolkit: https://grafana.com/docs/writers-toolkit/writing-guide/markdown-guide/#unordered-lists

Reply:
@MattDodsonEnglish You've made a good point. I changed to the * from the - because of a comment noting that the unordered lists created with a hyphen can cause an issue with either Prettier or the linter. I have no problem either way.